### ISSUE TITLE
Docker compose: Comment out "grafana" and "influxdb" custom option by default

### ISSUE THREAD

**Issue by isparkes** [blockchain-monitoring/blockchain-monitoring#21]
Hey! Great work! I would suggest to comment out the advanced options for "grafana" and "influxdb" in the exampe compose file by default. Leave them there for later, but they caused directories to be created, instead of files, leading to error

`"ERROR: for monitoring  Cannot start service monitoring: oci runtime error: container_linux.go:262: starting container process caused "process_linux.go:339: container init caused \"rootfs_linux.go:57: mounting \\\"/data/sdb/hl-factory/blockchain-monitoring/influxdb.conf\\\" to rootfs \\\"/var/lib/docker/aufs/mnt/c2d88ab78e92d245467b62da92987371ee9bc83fe450afa5e65da9cd628c5b44\\\" at \\\"/var/lib/docker/aufs/mnt/c2d88ab78e92d245467b62da92987371ee9bc83fe450afa5e65da9cd628c5b44/etc/influxdb/influxdb.conf\\\" caused \\\"not a directory\\\"\"""`

People who need to customise these will know where to look. Just a suggestion...

**Comment by rafizanbaharum** [comment]
I'm seeing this error with grafana and influxdb:
ERROR: for 25381cadba99_25381cadba99_25381cadba99_25381cadba99_25381cadba99_25381cadba99_25381cadba99_25381cadba99_25381cadba99_25381cadba99_25381cadba99_25381cadba99_blockchain-monitoring  Cannot start service monitoring: oci runtime error: container_linux.go:262: starting container process caused "process_linux.go:339: container init caused \"rootfs_linux.go:57: mounting \\\"/root/blockchain/blockchain-monitoring/influxdb.conf\\\" to rootfs \\\"/var/lib/docker/overlay/eefbd9443339cc1204a0b69992c9f459105a1f591989475be1bcbf84d3f34109/merged\\\" at \\\"/var/lib/docker/overlay/eefbd9443339cc1204a0b69992c9f459105a1f591989475be1bcbf84d3f34109/merged/etc/influxdb/influxdb.conf\\\" caused \\\"not a directory\\\"\""
: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

ERROR: for monitoring  Cannot start service monitoring: oci runtime error: container_linux.go:262: starting container process caused "process_linux.go:339: container init caused \"rootfs_linux.go:57: mounting \\\"/root/blockchain/blockchain-monitoring/influxdb.conf\\\" to rootfs \\\"/var/lib/docker/overlay/eefbd9443339cc1204a0b69992c9f459105a1f591989475be1bcbf84d3f34109/merged\\\" at \\\"/var/lib/docker/overlay/eefbd9443339cc1204a0b69992c9f459105a1f591989475be1bcbf84d3f34109/merged/etc/influxdb/influxdb.conf\\\" caused \\\"not a directory\\\"\""
: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
ERROR: Encountered errors while bringing up the project.


How do I remove grafana and influxdb options?

**Comment by rafizanbaharum** [comment]
Got it, just remove it from the yaml file. Thanks.

**Comment by isparkes** [comment]
Right! Once you have something to put in the files, you can re-enable them. I got around it by just touching the files and leaving them empty, but it's much more logical to comment out the lines.
